docs/content/stable/develop/quality-of-service/write-heavy-workloads.md

```diff
@@ -23,66 +24,59 @@ The idea is to slow down incoming writes to the speed that the database can hand
 ## Identifying reasons for write stalls
 
 Write stalls can occur for the following reasons:
+
 * Low CPU environments
 * Disks with low performance
 
 The following symptoms occur at the database layer:
 
 ### Flushes cannot keep up
+
 More writes than the system can handle could result in too many memtables getting created, which are queued up to get flushed. This puts the system in a suboptimal state, because a failure would need a large rebuild of data from WAL files, as well as requiring too many compactions for the system to catch up to a healthy state.
 
 ### Compaction cannot keep up
+
 The database getting overloaded with writes can also result in compactions not being able to keep up. This causes the SST files to pile up, degrading read performance significantly.
 
 ### WAL writes are too slow
-If WAL writes are slow, then writes experience higher latency which creates a natural form of admission control. The frequency of data getting synced to disk is controlled by the `durable_wal_write` flag. Note that fsync is disabled by default since YugabyteDB is expected to run in a Replication Factor 3 mode where each shard/tablet is replicated onto 3 independent fault domains such as hosts, availability zones, regions or clouds. If enabled, every single write into DocDB, YugabyteDB’s underlying document store, will be synchronized to disk before its execution is deemed successful. So, although there will be an increase in safety, there will be a hit to performance.
+
+If WAL writes are slow, then writes experience higher latency which creates a natural form of admission control. The frequency of data getting synced to disk is controlled by the `durable_wal_write` flag. Note that `fsync` is disabled by default as YugabyteDB is expected to run in a Replication Factor 3 mode where each shard/tablet is replicated onto 3 independent fault domains such as hosts, availability zones, regions, or clouds. If enabled, every single write into DocDB, YugabyteDB's underlying document store, will be synchronized to disk before its execution is deemed successful. So, although there will be an increase in safety, there will be a hit to performance.
```

Comment:
If WAL writes are slow, then writes experience higher latency which creates a natural form of admission control. The frequency of data getting synchronized to disk is controlled by the `durable_wal_write` flag. Note that `fsync` is disabled by default, as YugabyteDB is expected to run in a Replication Factor of 3 mode where each tablet is replicated onto three independent fault domains such as hosts, availability zones, regions, or clouds. If enabled, every single write into DocDB, which is a YugabyteDB's underlying document store, will be synchronized to disk before its execution is deemed successful. This means that even though there will be an increase in safety, performance will be negatively affected.